tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -540,6 +540,34 @@ def test_log_softmax_invalid(device, a_shape, axis, float_dtype):
         return chainerx.log_softmax(a, axis=axis)
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@pytest.mark.parametrize('input', [
+    numpy.asarray(0.), numpy.asarray(-1.), numpy.asarray(1.), numpy.asarray(
+        10.), numpy.asarray(float('inf')), numpy.asarray(float('nan')),
+    numpy.full((), 2.), numpy.full((0,), 2.), numpy.full((2, 3), 2.)
+])
+@pytest.mark.parametrize('contiguous', [None, 'C'])
+class testSigmoid(op_utils.NumpyOpTest):
```

Comment:
Can you rename the class as follows? It will otherwise raise an error if you sync with the latest master due to https://github.com/chainer/chainer/pull/6507.
```suggestion
class TestSigmoid(op_utils.NumpyOpTest):
```